app/uk/gov/hmrc/thirdpartyapplication/domain/models/ApplicationUpdate.scala

```diff
@@ -30,11 +30,14 @@ trait GatekeeperApplicationUpdate extends ApplicationUpdate {
 }
 
 case class ChangeProductionApplicationName(instigator: UserId, timestamp: LocalDateTime, gatekeeperUser: String, newName: String) extends GatekeeperApplicationUpdate
+case class ChangeProductionApplicationPrivacyPolicyLocation(instigator: UserId, timestamp: LocalDateTime, oldLocation: PrivacyPolicyLocation, newLocation: PrivacyPolicyLocation) extends ApplicationUpdate
```

Comment:
This may be a stupid question - but is this limited to production apps only?

Reply:
For this ticket yes, it only covers prod apps - sandbox apps use a different page in TPDFE which uses a different TPA endpoint. We don't want audit events raising for sandbox app updates, but even so it's an odd way to do things...

Reply:
OK